docs/rules/expect-expect.md

```diff
@@ -1,8 +1,15 @@
 # Enforce assertion to be made in a test body (`expect-expect`)
 
+<!-- RULE_NOTICE -- Generated by `yarn tools:regenerate-docs` -->
+
+<!-- prettier-ignore -->
+💼 This rule is enabled in the following [configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations): `all`.
+
+<!-- /RULE_NOTICE -->
+
 Ensure that there is at least one `expect` call made in a test.
 
-## Rule details
+## Rule Details
```

Comment:
I think we should actually be ensuring `details` is lowercase because that's more consistent with other headers (and technically more grammatically correct)

Reply:
Sounds good, updated.